### ISSUE TITLE
Overview of protected contents

### ISSUE THREAD

**Issue by pferreir** [indico/indico#5090]
Several users have requested some sort of "overview" of elements of an event whose protection differs from the event itself. This would allow them to quickly see which bits of the event are restricted to each users/groups, or purposefully made public.
This could be as simple as a (hierarchical) list of sessions/contributions/etc... which contains only the "diverging" contents.

**Comment by ThiefMaster** [comment]
We do have something like this in the management area. Not particularly structured, but at least a way to see everything that doesn't just inherit its protection:

![image](https://user-images.githubusercontent.com/179599/133589006-10859f2c-c491-4aa3-91be-8cbdd4b2e9d1.png)

**Comment by pferreir** [comment]
Yeah, we could probably build on that, maybe a standalone page would be better.
